cmd/olfullnode/devnet.go

```diff
@@ -85,6 +86,8 @@ func init() {
 	initCmd.AddCommand(testnetCmd)
 	testnetCmd.Flags().IntVar(&testnetArgs.numValidators, "validators", 4, "Number of validators to initialize devnet with")
 	testnetCmd.Flags().IntVar(&testnetArgs.numNonValidators, "nonvalidators", 0, "Number of non-validators to initialize the devnet with")
+	testnetCmd.Flags().IntVar(&testnetArgs.numWitness, "witness", 0, "number of witness to initialize mainnet with")
```

Comment:
Im sorry, but Im totally confused by the usage of the word witness... I feel this is not easily extendable to other chains like BTC or even any ERC20 token. With these changes, I sort of expected the addition of new chains would be a breeze and doesn't need a lot of work again changing a lot of these files.